clean_orig.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
@srirajpaul also good with deleting this file.

Reply:
Deleted.